src/pages/Home/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Button, TextField } from '@material-ui/core';
+import './style.css';
+import axios from 'axios';
+
+const Home = () => {
+    const [username, setUsername] = useState("");
+    const [error, setError] = useState(false);
+
+    async function getData(user){
+        try {
+            const { data } = await axios.get(`https://api.github.com/users/${user}/repos`);
+            setError(false);
+            window.location.pathname = `/${username}`;
+        } catch (error) {
+            setError('Unable to find GitHub user, please try again.');
```

Comment:
Cool that you handled the error here, when we did it we checked on the next page so we didn't have to fetch more than once but this is nicer I think

Reply:
Realised that because the error is handled here, if I just type an invalid username in the top bar it loads forever rather than using an error